study.md

```diff
@@ -29,21 +29,29 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
-```
+The event loop is responsible for invoking callbacks. The point of this is when
+async calls are completed, they return an event. The callbacks listen for the
+event and allow the function to continue.```
```

Comment:
Good, and the event loop will watch the call stack, when it is empty it will fire the next event from the queue.